Lib/html/parser.py

```diff
@@ -9,7 +9,6 @@
 
 
 import re
-import warnings
 import _markupbase
 
 from html import unescape
```

Comment:
I think this could be removed too since if it's no longer needed (even though removing it might break some code if people are doing `from html.parser import unescape` instead of `from html import unescape`).

Reply:
`unescape` is used in this module several times.